### DISCUSSION TITLE
Condition not editable in the custom code

### DISCUSSION THREAD

**Discussion by markosansa** [elementor/elementor#16480] [Developers Community]
Hi all, 
I have an issue but I'm not sure is something on my side, I would say an improvement is needed. 
When I create a custom code, after selecting the condition (example show in the page X Y) I'm not able to change the condition anymore which means I have to re-create the custom code again. 

Condition:
https://i.ibb.co/0ymT4dS/Screenshot-2021-10-05-at-12-24-03.png

Custom code:
https://i.ibb.co/rb120kJ/Screenshot-2021-10-05-at-12-24-20.png

Thank you.

**Comment by markosansa** [comment]
Sorry found the solution. 
Thank you

**Reply by Alfredopolodev** [reply]
How did you solve it?

**Reply by markosansa** [reply]
Make sure you have the updated version and no plugin conflicts.

**Comment by Alfredopolodev** [comment]
Cómo puedo ver que no haya conflictos con los complementos.

Alfredo Polo
666924332
***@***.***
